Trunk.toml

```diff
@@ -31,13 +31,21 @@ cargo = false
 # Proxies are only run as part of the `trunk serve` command.
 
 [[proxy]]
-# This proxy example has a backend and a rewrite field. Requests received on `rewrite` will be
-# proxied to the backend after rewriting the `rewrite` prefix to the `backend`'s URI prefix.
-# E.G., `/api/v1/resource/x/y/z` -> `/resource/x/y/z`
-rewrite = "/api/v1/"
-backend = "http://localhost:9000/"
+# This websocket proxy example has a backend and a path field. This example will listen for
+# websocket connections at `/api/ws` and proxy the those to `http://localhost:9000/api/ws`.
+# Websocket proxies must be listed before http ones
```

Comment:
```suggestion
# Websocket proxies must be listed before http ones.
```
Also, a question about this line. Why must WS proxies be listed first?

Reply:
This is because of how warp works. Warp goes from top to bottom in the filter list, if one matches it'll use that and discard everything else. Websocket filters are placed before http ones so they also need to come first in the config.